doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -1772,8 +1777,24 @@ In order to save hard disk space and avoid loss of the audit log records during
 The X-Road software does not offer special tools for archiving the audit log. The *rsyslog* can be configured to redirect the audit log to an external location.
 
 
-## 13 Back up and Restore
+## 13 Back up and restore
+
+It is possible to back up and later restore security server configuration.
+
+Backup archive contains
+- copy of serverconf database
+- user modifiable configuration files
+- TLS keys and certificates (both internal and ui certificate)
+- database credentials
 
+Note that backups contain information that must be kept secret (especially TLS keys and database credentials).
+In other words leaking this information could easily lead to full compromise of security server.
+It is therefore highly recommended that backup archives are encrypted and stored securely. Should the information still
+leak for whatever reason the security server should be considered as compromised and reinstalled from scratch.
+
+Starting from version 7.0 security server backups are signed and optionally encrypted. The GNU Privacy Guard \[[GnuPG](#Ref_GnuPG)\]
+is used for encryption and signing. GPG keypair is generated during security server initialisation. In addition to generated
+key additional public keys can be used to encrypt backups.
```

Comment:
I would change this part like this:

It is possible to back up and later restore security server configuration. A backup archive file contains the following configuration:

- copy of serverconf database
- user modifiable configuration files
- keys and certificates
  - security server's auth key and certificate
  - members' sign keys and certificates (that are stored in soft token)
  - security server's internal TLS key and certificate
  - security server's UI key and certificate
- database credentials.

Backups contain sensitive information that must be kept secret (for example, private keys and database credentials). In other words, leaking this information could easily lead to full compromise of security server. Therefore, it is highly recommended that backup archives are encrypted and stored securely. Should the information still leak for whatever reason the security server should be considered as compromised and reinstalled from scratch.

Security server backups are signed and optionally encrypted. The GNU Privacy Guard \[[GnuPG](#Ref_GnuPG)\] is used for encryption and signing. Security server's backup encryption key is generated during security server initialisation. In addition to the automatically generated backup encryption key, additional public keys can be used to encrypt backups.